src/components/Teams/Teams.js

```diff
@@ -27,7 +27,7 @@ class Teams extends Component {
           <>
             <h3 key={team.id}>{team.name}</h3>
               {team.members.map(member =>
-                <li>{member.firstname}
+                <li key={Math.random()}>{member.firstname}
```

Comment:
since it looks like line 26 maps through team, we should probably use team.id as the key

Reply:
quick fix is all, i didnt look over what is in team.members, if that had id's and such. team.id wouldnt work as thats before the mapping so wouldnt be unique

Reply:
member.id should work the line before is the map